homeassistant/components/device_tracker/__init__.py

```diff
@@ -398,15 +398,32 @@ def update(self):
 
 def load_config(path: str, hass: HomeAssistantType, consider_home: timedelta):
     """Load devices from YAML configuration file."""
+    dev_schema = vol.Schema({
```

Comment:
Partial validation is very important, since its a user editable file. (At least my view)

Not sure your approach will work for that?